Ecosystem Partners/getting-started-with-minecraft-pe-server-blueprint.md

```diff
@@ -0,0 +1,144 @@
+* Save money - Run your Minecraft PE server only when you use it.  You can host a server just in the evenings or weekends with your friends and not have to pay to run it 24/7 unless you chose to.  You can use the CenturyLink Cloud mobile app to start and stop your Minecraft server any time, any where.  
+* Fast Bandwidth - Your Minecraft server has a 2GB inbound network throughput ensuring low latency game play.
+* Easy backups - You can easily save your worlds anytime you choose
+* Customizable - Customize your Minecraft server by installing any mods you want!
+* No advanced technical experience required - Don't worry about setting up the VM and software dependencies, we did that for you.  Just run the Blueprint, add a public IP with firewall rules and start playing in minutes.
+* Parental Control - Control where your family plays and who they play with by having ownership of the server.
+* Secure - CenturyLink Cloud includes network security and allows you to block users from connecting to your server at a network level in addition to game whitelists.
+
+### Audience
+CenturyLink Cloud Users, Creepers and Minecrafters
+
+### Impact
+After reading this article, the user should feel comfortable getting started deploying and using the Minecraft PE Server Blueprint technology on CenturyLink Cloud.
+
+### Prerequisite
+- Access to the CenturyLink Cloud platform as an authorized user.
+- Confirm you are using Minecraft Personal Edition client
+  ![Minecraft PE Client Image](../images/ecosystem-minecraft-pe-server-1.png)
+
+### Install Minecraft PE Server on Linux Blueprint
+1. Locate the Mojang Minecraft PE Server Blueprint
+  1. Starting from the CenturyLink Control Panel, navigate to the Blueprints Library.
+  2. Search for “Minecraft PE” in the keyword search on the right side of the page.
+  3. Locate the 'Install Minecraft PE Server on Linux' Blueprint
+  ![Locate Minecraft PE Blueprint Image](../images/ecosystem-minecraft-pe-server-10.png)
+
+2. Choose and Deploy the Blueprint. Click the “Install Minecraft PE Server on Linux” Blueprint.
+  ![Deploy Minecraft PE Blueprint Image](../images/ecosystem-minecraft-pe-server-8.png)
+
+3. Configure the Blueprint with the following Minecraft specific information:
+  1. Server Name - Server Name, e.g. CLCMC
+  2. Server Description - Description of your server, e.g. Family Minecraft Server
+  3. Game Mode - Creative or Survival mode
+  4. minecraftpe user's password - Set a secure password for the minecraftpe user that runs the server software.
+  5. Click “next: step 2”
+  ![Configure Minecraft PE Blueprint Image](../images/ecosystem-minecraft-pe-server-11.png)
+
+4. Review, Confirm and Deploy the Blueprint.
+  1. Verify your configuration details.
+  2. Once verified, click on the ‘deploy blueprint’ button. This will kick off the blueprint deploy process and load a page to allow you to track the progress of the deployment.
```

Comment:
Please review all the single and double quotes and make sure they are not `straight quotes` editor like Microsoft Word convert them to a unicode character that show forward and backward direction. 

http://www.techrepublic.com/forums/questions/how-to-force-ms-word-to-use-plain-single-and-double-quotes/